core/src/main/java/org/elasticsearch/gateway/TransportNodesListGatewayStartedShards.java

```diff
@@ -128,7 +131,8 @@ protected NodeGatewayStartedShards nodeOperation(NodeRequest request) {
                 if (metaData != null) {
                     ShardPath shardPath = null;
                     try {
-                        shardPath = ShardPath.loadShardPath(logger, nodeEnv, shardId, metaData.getSettings());
+                        IndexSettings indexSettings = new IndexSettings(new Index(metaData.getIndex()), Settings.settingsBuilder().put(settings).put(metaData.getSettings()).build(), Collections.EMPTY_LIST);
```

Comment:
Okay, if it's intentional that's fine, just want to make sure we aren't adding things that were intentionally left out before.